docs/src/calling-contracts/custom-asset-transfer.md

```diff
@@ -0,0 +1,7 @@
+# Custom asset transfers
```

Comment:
```suggestion
# Custom assets transfer
```

Reply:
Filename should be `custom-assets-transfer` IMO

Reply:
How about custom asset transfer since it is a single transfer of a single asset